### DISCUSSION TITLE
Security Onion Console with just a Single Alerts application.

### DISCUSSION THREAD

**Discussion by paraghakke** [Security-Onion-Solutions/securityonion#6377] [General] [answered]
Hi Team,

I just want a user to have access to only Alerts dashboard and hide all other options in Security Onion Console, for that matter hide the side menu panel as well and display only Standalone Alerts Dashboard.

Have gone through various configuration files and also the tools.json file but did not find a solution to it.

Will be helpful if someone from team can guide with few options

Screen looking similar to the image attached.

Thanks
![AlertsConsole](https://user-images.githubusercontent.com/3206623/143887521-b165792b-5e2e-4822-9923-bb32cede3db8.JPG)
.

**Comment by dougburks** [comment, marked as answer]
I don't think there is an easy way to do this. I think you would have to create your own custom modifications which we don't support.
